resources/views/app/master.blade.php

```diff
@@ -39,6 +39,10 @@
         <script src = "{!! load_asset('/js/comment.js') !!}"></script>
 
         <script src="{!! load_asset('/js/socialmediashare.js') !!}"></script>
+        <script src="{!! load_asset('/js/commentupdate.js') !!}"></script>
+
+        <link href = "{!! asset('/library/sweetalert/sweetalert.css') !!}" rel="stylesheet"  />
+        <script type="text/javascript" src = "{!! asset('/library/sweetalert/sweetalert.min.js') !!}"></script>
```

Comment:
**Code Functionality - Nitpick**

Please replace `asset` with `load_asset` to ensure it sets HTTPS on production environment for referencing the scripts.